scripts/autorelease.sh

```diff
@@ -21,7 +21,7 @@ trap finish EXIT
 assets_dir=$TEMPDIR/assets
 
 # Build tezos-release
-nix-build . -A tezos-release -o "$TEMPDIR"/"$project" \
+nix-build . -A release -o "$TEMPDIR"/"$project" \
           --arg docker-binaries ./binaries/docker --arg docker-arm-binaries ./arm-binaries/docker
```

Comment:
Looks like it should be `arm-docker-binaries` instead of `docker-arm-binaries` now